DataFormats/HcalIsolatedTrack/src/classes_def.xml

```diff
@@ -2,7 +2,8 @@
 
   <class name="edm::Wrapper<std::map <int,std::pair<double,double> > >"/> 
 
-  <class name="reco::IsolatedPixelTrackCandidate" ClassVersion="13">
+  <class name="reco::IsolatedPixelTrackCandidate" ClassVersion="14">
+   <version ClassVersion="14" checksum="2494226563"/>
```

Comment:
When making the new DataFormat, re-use classversion 14 but with the new checksum, as 14 was not yet used anywhere.